src/arvgvinterface.c

```diff
@@ -71,8 +77,14 @@ arv_gv_discover_socket_set_buffer_size (ArvGvDiscoverSocket *discover_socket, gi
 	int result;
 
 	socket_fd = g_socket_get_fd (discover_socket->socket);
-
+#ifndef G_OS_WIN32
```

Comment:
There is 2 occurences of this code, it is a call for an helper function in arvnetwork.c

Reply:
added as `arv_socket_set_recv_buffer_size`, declared in `arvnetoworkprivate.h`.